regolith/builders/proposalreviewbuilder.py

```diff
@@ -0,0 +1,65 @@
+"""Builder for Current and Pending Reports."""
+import datetime
+import time
+from nameparser import HumanName
```

Comment:
I think this needs to be added to the requirements

Reply:
OK, I added to ``run.txt`` in ``requirements``.  Let me know if it should be removed.